core/src/com/unciv/models/ruleset/Difficulty.kt

```diff
@@ -14,7 +14,7 @@ class Difficulty: INamed {
     var policyCostModifier:Float = 1f
     var unhappinessModifier:Float = 1f
     var barbarianBonus:Float = 0f
-    var startingUnits = ArrayList<String>()
+    var startingUnits = ArrayList<String>() // I would like to rename these to 'bonus starting units' now that they also exist in Eras.json, but something something mod compatibility
```

Comment:
Au Contraire Snapeydoodle
Creating a new bonusStartingUnits would actually help differentiate the 'new' functionality from the 'old' functionality and help with the gradual deprecation!